### ISSUE TITLE
more gui not available for localization

### ISSUE THREAD

**Issue by ordtrogen** [Chia-Network/chia-blockchain-gui#108]
![Skärmavbild 2021-02-27 kl  15 32 23](https://user-images.githubusercontent.com/15184875/109390302-16de3b80-7911-11eb-87c1-b613a82ce3a4.png)

**Comment by jespino** [comment]
This is a server message error directly shown there. The server is not returning the structured data to detect the type of error and show this translated in the interface. So... or the server adds structured data identifying the error, or the server translates the message or we accept that server error messages are untranslated. @mariano54 @hoffmang9 What is the expectation about this?

**Comment by mariano54** [comment]
Good point. In the server rpcs, we usually throw a ValueError("random error message str"), which is caught by rpc server. 

Instead, we should probably throw one of the values in the Err class (src.util.error).

And in the rpc server we can catch that and turn it into an integer.

This might be kind of hard to do because there are many places where exceptions are thrown, in many parts of the code. But we can start with some, like this error.
